cmd/agent/main.go

```diff
@@ -1,3 +1,21 @@
 package main
 
-func main() {}
+import (
+	"log"
+
+	agent "github.com/VanGoghDev/practicum-metrics/internal/agent/app"
+	"github.com/VanGoghDev/practicum-metrics/internal/agent/config"
+)
+
+func main() {
+	cfg, err := config.Load()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
К ошибкам лучше добавлять их контекст:

```suggestion
		log.Fatalf("failed to load config: %w", err)
```